smswithoutborders_libsig/keystore.py

```diff
@@ -1,4 +1,10 @@
-"""A module for managing a secure keystore."""
+"""
+This program is free software: you can redistribute it under the terms
+of the GNU General Public License, v. 3.0. If a copy of the GNU General
+Public License was not distributed with this file, see <https://www.gnu.org/licenses/>.
+"""
+
+# A module for managing a secure keystore.
```

Comment:
Keep the existing file description at the top of the docstring.

```suggestion
"""
A module for managing a secure keystore.
 
This program is free software: you can redistribute it under the terms
of the GNU General Public License, v. 3.0. If a copy of the GNU General
Public License was not distributed with this file, see <https://www.gnu.org/licenses/>.
"""

```